Core/Graphics/Source/FrameBuffer.cpp

```diff
@@ -7,10 +7,10 @@ namespace Babylon::Graphics
 {
     namespace
     {
-        void setDefaultClearMode(bgfx::ViewId viewId, bgfx::FrameBufferHandle handle)
+        void setDefaultClearMode(bgfx::ViewId viewId, bgfx::FrameBufferHandle handle, uint16_t flags, uint32_t rgba, float depth, uint8_t stencil)
```

Comment:
nit: this isn't your change, but the style is wrong.. same for `setViewPort`

```suggestion
        void SetDefaultClearMode(bgfx::ViewId viewId, bgfx::FrameBufferHandle handle, uint16_t flags, uint32_t rgba, float depth, uint8_t stencil)
```